HPXMLtoOpenStudio/resources/hotwater_appliances.rb

```diff
@@ -226,7 +226,9 @@ def self.apply(model, runner, hpxml, weather, spaces, hot_water_distribution,
       t_mix = 105.0 # F, Temperature of mixed water at fixtures
       avg_setpoint_temp = 0.0 # WH Setpoint: Weighted average by fraction DHW load served
       hpxml.water_heating_systems.each do |water_heating_system|
-        avg_setpoint_temp += water_heating_system.temperature * water_heating_system.fraction_dhw_load_served
+        wh_setpoint = water_heating_system.temperature
+        wh_setpoint = Constants.WaterHeaterSetpointAverage if wh_setpoint.nil? # using detailed schedules
```

Comment:
Use `Waterheater.get_default_hot_water_temperature(eri_version)` instead?

Reply:
I was going to leave that comment. Yes, let's remove the new Constant and reference the default value instead.